commissioning/dest.env

```diff
@@ -0,0 +1,2 @@
+b=200
```

Comment:
Is this just example stuff?

Reply:
Whoops, this was before I successfully mocked out all the file io in the tests. Deleting.